nodecg-io-ws-client/extension/index.ts

```diff
@@ -10,6 +10,10 @@ interface WSClientServiceConfig {
 
 export interface WSClientServiceClient {
     getRawClient(): WebSocket;
+    send(message: string): void;
+    onMessage(func: (message: WebSocket.Data) => void): void;
+    onClose(func: () => void): void;
+    onError(func: () => Error): void;
```

Comment:
This is wrong as the callback gets the error from the WS and not the other way arround:
```suggestion
    onError(func: (error: Error) => void): void;
```
The function definition has it correctly: https://github.com/codeoverflow-org/nodecg-io/pull/61/files#diff-d108bf9dcfde44aecbdc8d34cbea7425R61